golden/golden.go

```diff
@@ -97,7 +91,7 @@ func Assert(t assert.TestingT, actual string, filename string, msgAndArgs ...int
 func String(actual string, filename string) cmp.Comparison {
 	return func() cmp.Result {
 		actualBytes := removeCarriageReturn([]byte(actual))
-		result, expected := compare(actualBytes, filename, removeCarriageReturn)
+		result, expected := compare(actualBytes, filename)
```

Comment:
We are already applying the normalize here before calling `compare`, so I don't think we need to pass in the normalize function.